src/Rvig.HaalCentraalApi.Personen/Services/GbaPersonenApiService.cs

```diff
@@ -163,12 +185,67 @@ await LogProtocolleringInDb(afnemerCode, result.Select(x => x.pl_id).ToList(),
 		return (new RaadpleegMetBurgerservicenummerResponse { Personen = result.ConvertAll(gbaPersoon => gbaPersoon.persoon) ?? new List<GbaPersoon>() }, result.Select(x => x.pl_id).ToList());
 	}
 
-	/// <summary>
-	/// Get personen via search params (child of PersonenQuery)
-	/// </summary>
-	/// <param name="model"></param>
-	/// <returns>Response object with list of restricted person data</returns>
-	private async Task<(List<T>? personen, List<long>? plIds)> GetPersonenBeperktBase<T>(PersonenQuery model, bool? inclusiefOverledenPersonen = false) where T : GbaPersoonBeperkt
+    private async Task<List<GbaPersoon>> GetGezagPersonen(List<string> gezagBsns)
+    {
+        (IEnumerable<(GbaPersoon persoon, long pl_id)>? personenData, int _) = await _getAndMapPersoonService.GetPersonenMapByBsns(gezagBsns, null, new List<string> { "naam", "geslacht", "geboorte.datum" }, _protocolleringAuthorizationOptions.Value.UseAuthorizationChecks);
+
+        List<GbaPersoon> gezagPersonen = new();
+        if (personenData != null)
+        {
+            gezagPersonen = personenData.Select(x => x.persoon).ToList();
+        }
+
+		return gezagPersonen;
+    }
+
+    private static List<string> GetGezagBsns(List<ApiModels.Gezag.AbstractGezagsrelatie> gezagsrelaties)
```

Comment:
ik denk dat het handig zou zijn om voor nieuwe code unit tests toe te voegen. Anders krijgen we de code coverage nooit omhoog